backend/tracim_backend/lib/core/url_preview.py

```diff
@@ -39,25 +44,69 @@ def __init__(self, title: Optional[str], description: Optional[str], image: Opti
         self.image = image
 
 
+class URLPreviewResponse(LengthLimitedResponse):
```

Comment:
As in the webpreview's fork changes that I reviewed, I do not think using a class here is needed. And I still do not see the use of inheritance from `requests.Response`.